### PR TITLE
Aaq step2

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by LeoMcA** [comment]
The automatic reformatting of the templates makes it almost impossible to see what's actually changed, can you turn this off in your editor?

**Comment by akatsoulas** [comment]
That's why I separated the PR into 2 commits. The second commits holds the changes and the first one is just formatting.

**Comment by LeoMcA** [comment]
> That's why I separated the PR into 2 commits. The second commit holds the changes and the first one is just formatting.

ah! I hadn't clocked that, apologies

File: kitsune/products/jinja2/products/includes/topic_macros.html [hunk 1]
```diff
@@ -42,11 +42,9 @@ <h3 class="card--title">
         {{ _('Still need help?') }}
       </h3>
       <p class="card--desc">{{ _('We’re here for you. Post a question to our support forums and get answers from our community of experts.') }}</p>
-      <a
-        class="sumo-button primary-button button-lg"
-        href="{{ url('questions.aaq_step1') }}"
-        data-event-category="link click"
-        data-event-action="topic"
+      <a class="sumo-button primary-button button-lg" {% if product %}
+        href="{{ url('questions.aaq_step2', product_key=product_key)}}" {%else %}
```
Comment (LeoMcA) [inline kitsune/products/jinja2/products/includes/topic_macros.html:46]: Should be `{% else %}`

nit: might be cleaner to do the if/else above, and `set` a variable which we then use as the value of the href attribute, like:
```
{% if product %}
  {% set aaq_url = ... %}
{% else %}
  {% set aaq_url = ... %}
{% endif %}
<a href="{{ aaq_url }}" ...

File: kitsune/products/views.py [hunk 2]
```diff
@@ -18,12 +19,20 @@ def product_list(request):
     return render(request, template, {"products": products})
 
 
+def _get_aaq_product_key(products, slug):
```
Comment (LeoMcA) [inline kitsune/products/views.py:22]: nit: I'd just take a single slug argument here, and directly reference `aaq_config.products` in the function

File: kitsune/products/views.py [hunk 3]
```diff
@@ -18,12 +19,20 @@ def product_list(request):
...
+    product_key = ""
```
Comment (LeoMcA) [inline kitsune/products/views.py:23]: nit: IMO `None` would make more sense here

**Review by LeoMcA — APPROVED** [review]
One small formatting typo, and a few optional/opinionated nits. Nice!

**Pull request merged**